### ISSUE TITLE
My experience on running the model

### ISSUE THREAD

**Issue by beautifulSun** [richzhang/colorization#56]
It is the first time i have used Caffe to build my development environment,so there are some difficulties in running this model.Luckily,it worked.Now share some experience,hope it is useful.
--------------------------------------------------------------------------------------------------------
My configuration:
ubuntu16.04,GTX1080,opencv3.4.0,the latest version of Caffe.
--------------------------------------------------------------------------------------------------------
I think the most important is caffe installation.
1.git clone -b master --single-branch https://github.com/richzhang/colorization.git
you will get colorization folder.but ./colorization  and  ./colorization/colorization are the same content,you can only use one of them.
2.git clone https://github.com/BVLC/caffe.git
3.the last layer of this model is SoftmaxCrossEntropyLayer which is not defined in Caffe.So in folder ./colorization/resources,softmax_cross_entropy_loss_layer.cpp,softmax_cross_entropy_loss_layer.cu need to be moved to caffe/src/caffe/layers/,softmax_cross_entropy_loss_layer.hpp should be moved to caffe/include/caffe/layers/,then compiling Caffe. gedit ~/.bashrc to configure pythonpath.
4.About lmdb file,please refer to
https://stackoverflow.com/questions/31427094/a-guide-to-convert-imageset-cpp#
5.run the model.(Don't run  ./train/fetch_caffe.sh).

**Comment by Mennat-Ullah** [comment]
i have problem on  lmdb file , if you can help me please ...
i want to use voc (small data , faster train) but i can't create lmdb like imagenet lmdb
any help ?

**Comment by beautifulSun** [comment]
You can refer to https://stackoverflow.com/questions/31427094/a-guide-to-convert-imageset-cpp#
if for colorization,train.txt/test.txt does not need add the label
then change the training/test set path to the location of the LMDB file on your machine

**Comment by beautifulSun** [comment]
> i have problem on lmdb file , if you can help me please ...
> i want to use voc (small data , faster train) but i can't create lmdb like imagenet lmdb
> any help ?

You can refer to https://stackoverflow.com/questions/31427094/a-guide-to-convert-imageset-cpp#
if for colorization,train.txt/test.txt does not need add the label
then change the training/test set path to the location of the LMDB file on your machine

**Comment by Mennat-Ullah** [comment]
thanks
